lib/ui/pages/location_page.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:food_app/utils/app_color.dart';
+import 'package:food_app/utils/app_typography.dart';
+
+class LocationPage extends StatelessWidget {
+  const LocationPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        body: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 20,horizontal: 22),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Image.asset("assets/images/binocular girl.png"),
```

Comment:
we can change the space in the file name with the underscore `_` and it will be more proper `binocular_girl.png`